docs/src/content/docs/fr/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,295 @@
+    module.exports = {
+    	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    	plugins: [starlightPlugin()],
+    };
+    ```
+
+### Mettre en forme Starlight avec Tailwind
+
+Starlight utilise les valeurs de votre [configuration de thème Tailwind](https://tailwindcss.com/docs/theme) dans son interface utilisateur.
+
+Si définies, les options suivantes remplaceront les styles par défaut de Starlight :
+
+- `colors.accent` — utilisé pour les liens et la mise en évidence de l'élément courant
+- `colors.gray` — utilisé pour les couleurs d'arrière-plan et les bordures
+- `fontFamily.sans` — utilisé pour le texte de l'interface utilisateur et du contenu
+- `fontFamily.mono` — utilisé pour les exemples de code
+
+```js {13,16,21,24}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Votre couleur d'accentuation préférée.
+				// Indigo est la plus proche des valeurs par défaut de Starlight.
+				accent: colors.indigo,
+				// Votre échelle de gris préférée.
+				// Zinc est la plus proche des valeurs par défaut de Starlight.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				// Votre police de texte préférée.
+				// Starlight utilise une pile de polices système par défaut.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Votre police de code préférée.
+				// Starlight utilise des polices système à chasse fixe par défaut.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Personnalisation du thème
+
+Le thème de couleur de Starlight peut être contrôlé en remplaçant ses propriétés personnalisées par défaut.
+Ces variables sont utilisées dans toute l'interface utilisateur avec une gamme de nuances de gris utilisées pour les couleurs de texte et d'arrière-plan et une couleur d'accentuation utilisée pour les liens et pour mettre en évidence les éléments courants durant la navigation.
+
+### Editeur de thème de couleur
+
+Utiliser les contrôles ci-dessous pour modifier les palettes de couleurs d'accentuation et de gris de Starlight.
+Les zones d'aperçu sombre et clair afficheront les couleurs résultantes, et la page entière sera également mise à jour pour prévisualiser vos modifications.
+
+Quand vous êtes satisfait de vos modifications, copiez le code CSS ou Tailwind ci-dessous et utilisez-le dans votre projet.
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Préréglages',
+			ocean: 'Océan',
+			forest: 'Forêt',
+			oxide: 'Oxyde',
+			nebula: 'Nébuleuse',
+			default: 'Par défaut',
+			random: 'Aléatoire',
+		},
+		editor: {
+			accentColor: 'Accentuation',
+			grayColor: 'Gris',
+			hue: 'Teinte',
+			chroma: 'Chroma',
```

Comment:
An official reference for keeping "Chroma" https://fr.wikipedia.org/wiki/Chrominance